app/src/main/java/com/example/next_show/data/RecommendationClient.java

```diff
@@ -57,7 +87,7 @@ public void onFailure(Call<List<com.uwetrottmann.trakt5.entities.Show>> call, Th
     }
 
     // get it by genre from recommended shows call
-    public void fetchGenreMatchedShows(Shows traktShows, ResponseCallback callback) {
+    private void fetchGenreMatchedShows(Shows traktShows, ResponseCallback callback) {
```

Comment:
I'm not sure why you need to call this in the other fetches.  It seems like this code should just know how to fetch the different shows.  If you get an empty response from one of them, then in the feedback fragment, you would handle doing additional fetching.

I don't think we want to do this condition logic where we sometimes chain this call in other data fetching calls